extensions/src/main/java/org/pentaho/platform/web/http/filters/PentahoWebContextFilter.java

```diff
@@ -158,6 +160,16 @@ public void doFilter( ServletRequest request, ServletResponse response, FilterCh
         printReservedCharsDisplay( out );
         printReservedRegexPattern( out );
 
+        boolean noOsgiRequireConfig = "true".equals( request.getParameter( "noOsgiRequireConfig" ) );
+        if ( !noOsgiRequireConfig ) {
+          final String useFullyQualifiedUrlParameter = httpRequest.getParameter( "fullyQualifiedUrl" );
```

Comment:
Don' see why not let the value determined according to local logic, `shouldUseFullyQualifiedUrl`, pass through.